tests/test_content_manifest.py

```diff
@@ -267,71 +272,6 @@ def test_to_json(mock_top_level_purl, app, package, subpath):
         mock_top_level_purl.assert_called_once_with(request, subpath=subpath)
 
 
-@pytest.mark.parametrize(
-    "package, internal_attr, internal_data",
-    [
-        (
-            {"name": "grc-ui", "type": "npm", "version": "1.0.0"},
-            "_npm_data",
-            # The id of the mock Package is 1, the purl is also mocked
-            {1: {"purl": "mock-package-purl", "sources": [], "dependencies": []}},
-        ),
-        (
-            {"name": "requests", "type": "pip", "version": "2.24.0"},
-            "_pip_data",
-            {1: {"purl": "mock-package-purl", "sources": [], "dependencies": []}},
-        ),
-        (
-            {"name": "grc-ui", "type": "yarn", "version": "1.0.0"},
-            "_yarn_data",
-            {1: {"purl": "mock-package-purl", "sources": [], "dependencies": []}},
-        ),
-        (
-            {"name": "example.com/org/project", "type": "go-package", "version": "1.1.1"},
-            "_gopkg_data",
-            # go-package is special, it also gets "name"
-            {
-                1: {
-                    "name": "example.com/org/project",
-                    "purl": "mock-package-purl",
-                    "sources": [],
-                    "dependencies": [],
-                },
-            },
-        ),
-        (
-            {"name": "example.com/org/project", "type": "gomod", "version": "1.1.1"},
-            "_gomod_data",
-            # gomod is special, it is only used to finalize go-package data
-            {"example.com/org/project": {"purl": "mock-package-purl", "dependencies": []}},
-        ),
-    ],
-)
-@mock.patch("cachito.web.models.Package.to_top_level_purl")
-# set_go_package_sources must be mocked because it is destructive towards _gopkg_data
-@mock.patch("cachito.web.content_manifest.ContentManifest.set_go_package_sources")
-def test_to_json_properly_sets_internal_data(
```

Comment:
This test was removed because the `test_process_*` tests are now actually using the to_json logic to function (making it redundant).